pages.bn/common/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
```

Comment:
This transaction isn't in sync with the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/cd.md). Add the missing command and reorder it in correct order.